packages/arb-avm-cpp/avm_values/src/vmValueParser.cpp

```diff
@@ -140,6 +140,9 @@ Operation operation_from_json(const nlohmann::json& op_json,
         std::cerr << "Invalid opcode " << opcode_json << "\n";
     }
     auto opcode = opcode_json.get<OpCode>();
+    if (!instructionValidity()[(size_t)opcode]) {
```

Comment:
Opcodes that aren't defined should still be valid inputs, but will cause an error when executed. I think we probably want to allow the parser to read in invalid opcodes like this and probably print a warning to `cerr` instead of failing